hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,111 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "{job_prefix} qrun -optimize"
+  run_cmd:    "{job_prefix} qrun -simulate"
```

Comment:
Move the `-optimize` and `-simulate` to `build_opts` and `run_opts` respectively.

Reply:
Also, maybe reference these binaries with `QUESTA_HOME` variable? Else, add a step in the install user guide to add `$QUESTA_HOME/bin` to the `$PATH` variable.

Reply:
`qrun -optimize` and  `qrun -simulate` are the newer version of `vlog` and `vsim`, so are not really build options. Moving -simulate to run_opts breaks gui simulations because the -simulate argument is pushed way back in the qrun command string unfortunately. I could move the `-optimize` but I think I will leave them as they are if you don't mind.
```
build_cmd:  "{job_prefix} qrun"
run_cmd:    "{job_prefix} qrun -simulate"
```
seems a bit confusing to me.

Sure, good shout with `QUESTA_HOME`